platforms/gke/base/core/container_cluster/container_cluster.tf

```diff
@@ -267,10 +218,57 @@ resource "google_container_cluster" "cluster" {
   }
 }
 
-data "google_container_cluster" "default" {
-  depends_on = [google_container_cluster.cluster]
+resource "google_container_node_pool" "system" {
+  # Variables
+  cluster            = google_container_cluster.cluster.name
+  initial_node_count = 1
+  location           = var.cluster_region
+  name               = "system"
```

Comment:
Is it possible to have a variable here in order to customize the name? It will be useful for the federated learning use case. Thanks